# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1502380094 14400
#      Thu Aug 10 11:48:14 2017 -0400
# Node ID 8f2d51b6235f5a34eb5beed622f596af29333467
# Parent  3c5a49805dbdfa39c32a58583431a59f2078deab
* pager.cc (default_pager): Use config::default_pager instead of build_env::DEFAULT_PAGER.

diff --git a/libinterp/corefcn/pager.cc b/libinterp/corefcn/pager.cc
--- a/libinterp/corefcn/pager.cc
+++ b/libinterp/corefcn/pager.cc
@@ -28,17 +28,16 @@ along with Octave; see the file COPYING.
 #include <iostream>
 #include <string>
 
 #include "cmd-edit.h"
 #include "oct-env.h"
 #include "oct-syscalls.h"
 #include "singleton-cleanup.h"
 
-#include "build-env.h"
 #include "defaults.h"
 #include "defun.h"
 #include "error.h"
 #include "errwarn.h"
 #include "input.h"
 #include "octave.h"
 #include "ovl.h"
 #include "pager.h"
@@ -61,17 +60,17 @@ static std::string diary_file ("diary");
 static std::ofstream external_diary_file;
 
 static std::string
 default_pager (void)
 {
   std::string pager_binary = octave::sys::env::getenv ("PAGER");
 
   if (pager_binary.empty ())
-    pager_binary = octave::build_env::DEFAULT_PAGER;
+    pager_binary = octave::config::default_pager ();
 
   return pager_binary;
 }
 
 // The shell command to run as the pager.
 static std::string VPAGER = default_pager ();
 
 // The options to pass to the pager.
